doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -184,18 +185,246 @@ if transceiver is not present:
+        SfpStateUpdateTask ->> APPL_DB: PORT_TABLE:<lport>.MEDIA_SETTINGS_SYNC_STATUS = <br> MEDIA_SETTINGS_NOTIFIED
+        activate OA
+        APPL_DB -->> OA: Notify media settings for ports
+        Note over OA: Disable admin status<br>setPortSerdesAttribute
+        OA ->> APPL_DB: PORT_TABLE:<lport>.MEDIA_SETTINGS_SYNC_STATUS = MEDIA_SETTINGS_DONE
+        Note over OA: initHostTxReadyState
+        deactivate OA
+    end
+```
+
+## XCVRD termination during syncd/swss/orchagent crash
+
+The below sequence diagram captures the termination of XCVRD during syncd/swss/orchagent crash.
+<br> supervisord will respawn XCVRD after termination as xcvrd is killed using SIGABRT signal
+
+```mermaid
+sequenceDiagram
+    participant OA
+    participant APPL_DB
+    participant XCVRDMT as XCVRD main thread
+    participant CmisManagerTask
+    participant DomInfoUpdateTask
+    participant SfpStateUpdateTask
+
+    activate OA
+    activate XCVRDMT
+    activate CmisManagerTask
+    activate DomInfoUpdateTask
+    activate SfpStateUpdateTask
+    OA -x OA: Crashes while handling a routine
+    deactivate OA
+    OA ->> APPL_DB : DEL PORT_TABLE
+
+    XCVRDMT -x APPL_DB : XCVRD main thread proecesses DEL event of APPL_DB PORT_TABLE
+    Note over XCVRDMT: generate_sigabrt = True
+    alt If threads > 0 are dead
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGKILL
+    end
+    XCVRDMT -x CmisManagerTask : Stop CmisManagerTask
+    deactivate CmisManagerTask
+    XCVRDMT -x DomInfoUpdateTask : Stop DomInfoUpdateTask
+    deactivate DomInfoUpdateTask
+    XCVRDMT -x SfpStateUpdateTask : Stop SfpStateUpdateTask
+    deactivate SfpStateUpdateTask
+    Note over XCVRDMT : deinit()
+    alt self.sfp_error_event.is_set()
+        XCVRDMT -x XCVRDMT : sys.exit(SFP_SYSTEM_ERROR)
+    else if generate_sigabrt is True
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGABRT
+
+    else
+        XCVRDMT -x XCVRDMT : Graceful exit
+    end
+    deactivate XCVRDMT
+```
+
+## Test plan and expectation
+|       Event      | APPL_DB cleared | Xcvrd restarted | Media renotify | MEDIA_SETTINGS_SYNC_STATUS value on   xcvrd boot-up for initialized transceiver | CMIS re-init triggered | Link flap |
+|:----------------:|:---------------:|:---------------:|:--------------:|:-----------------------------------------------------------------------------:|:----------------------:|:---------:|
+| Xcvrd restart    | N               | Y               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | N         |
+| Pmon restart     | N               | Y               | N              | MEDIA_SETTINGS_DONE                                                           | N                      | N         |
+| Swss restart     | Y               | Y               | Y              | MEDIA_SETTINGS_DEFAULT                                                        | Y                      | Y         |
```

Comment:
this is not a case of link flap per se.
All critical SW components went down along with HW ones (NPU, data-path power zone etc.) so its equivalent to SW/NOS restart.
We should mark this case as N/A under this link flap column